extension/jaegerremotesampling/internal/grpc_test.go

```diff
@@ -63,29 +63,44 @@ func TestSamplingGRPCServer_Shutdown(t *testing.T) {
 			grpcServer: &grpcServerMock{
 				timeToGracefulStop: time.Minute,
 			},
-			timeout: 5 * time.Second,
-		},
-		{
-			name:    "grpc server not started",
-			timeout: time.Minute,
-			expect:  errGRPCServerNotRunning,
+			timeout: time.Millisecond,
 		},
 	}
 
 	for _, tc := range tt {
 		t.Run(tc.name, func(t *testing.T) {
 			srv := &SamplingGRPCServer{grpcServer: tc.grpcServer}
 			ctx, cancel := context.WithTimeout(context.Background(), tc.timeout)
+			tc.grpcServer.Serve(nil)
```

Comment:
```suggestion
			assert.NoErorr(t, tc.grpcServer.Serve(nil))
```
Lint requires the return value to be checked here.

Reply:
Yes. I still have to fix the CI. I made the test pass and pushed (Friday night :D).

Thanks for the hint!